conformance/google-java/src/test/kotlin/com/connectrpc/conformance/Conformance.kt

```diff
@@ -383,22 +389,49 @@ class Conformance(
     @Test
     fun unimplementedService(): Unit = runBlocking {
         val countDownLatch = CountDownLatch(1)
-        unimplementedServiceClient.unimplementedCall(Empty.newBuilder().build()) { response ->
+        unimplementedServiceClient.unimplementedCall(empty {}) { response ->
             assertThat(response.code).isEqualTo(Code.UNIMPLEMENTED)
             countDownLatch.countDown()
         }
         countDownLatch.await(500, TimeUnit.MILLISECONDS)
         assertThat(countDownLatch.count).isZero()
     }
 
+    @Test
+    fun unimplementedServerStreamingService(): Unit = runBlocking {
```

Comment:
Added this test which caught the streaming behavior issue: https://github.com/connectrpc/connect-kotlin/pull/101#discussion_r1327842292